frontend/src/scenes/session-recordings/playlist/SessionRecordingsPlaylistSettings.tsx

```diff
@@ -20,16 +20,71 @@ export function SessionRecordingsPlaylistSettings(): JSX.Element {
-                        onChangeEventDescription="session recording list duration type to show selected"
-                    />
-                </div>
-            )}
+            <div className="flex flex-row items-center justify-between space-x-2">
+                <span className="text-black font-medium">Random sample</span>
+                <LemonSwitch
+                    aria-label="Random sample"
+                    checked={randomSample}
+                    onChange={() => setRandomSample(!randomSample)}
+                    tooltip="Chooses a random sample of recordings matching the filters"
+                />
+            </div>
+            <div className="flex flex-row items-center justify-between space-x-2">
+                <span className="text-black font-medium">Order by</span>
+                <LemonSelect
+                    options={[
+                        {
+                            value: 'start_time',
+                            label: 'Latest',
+                        },
+                        {
+                            label: 'Longest',
+                            options: [
+                                {
+                                    value: 'duration',
+                                    label: 'Total duration',
+                                },
+                                {
+                                    value: 'active_seconds',
+                                    label: 'Active duration',
+                                },
+                                {
+                                    value: 'inactive_seconds',
+                                    label: 'Inactive duration',
+                                },
+                            ],
+                        },
+                        {
+                            label: 'Most active',
+                            options: [
+                                {
+                                    value: 'click_count',
+                                    label: 'Clicks',
+                                },
+                                {
+                                    value: 'keypress_count',
+                                    label: 'Key presses',
+                                },
+                                {
+                                    value: 'mouse_activity_count',
+                                    label: 'Mouse activity',
+                                },
+                            ],
+                        },
+                        {
+                            value: 'console_error_count',
+                            label: 'Most errors',
+                        },
+                        {
+                            value: 'random_sample',
+                            label: 'Random sample',
+                        },
+                    ]}
+                    size="small"
+                    value={orderBy as string}
```

Comment:
could probably avoid this cast by properly typing the options but they're nested and i'm lazy 🤣